### PR TITLE
Integration into OSS-Fuzz

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Hi

I have integrated fuzzing into spotify-json and was thinking that it would be nice to set up continuous fuzzing by way of OSS-Fuzz. In this PR: https://github.com/google/oss-fuzz/pull/5014 I have done exactly that, namely created the necessary logic from an OSS-Fuzz perspective to integrate. In that PR you also see the fuzzer attached, but it would be nicer to have the fuzzer integrated into the upstream project itself rather than keeping it in OSS-Fuzz - so if this PR gets merged I will remove it from OSS-Fuzz

Essentially, OSS-Fuzz is a free service run by Google that performs continuous fuzzing of important open source projects. The only expectation of integrating into OSS-Fuzz is that bugs will be fixed. This is not a "hard" requirement in that no one enforces this and the main point is if bugs are not fixed then it is a waste of resources to run the fuzzers, which we would like to avoid.

If you would like to integrate, could I please have an email(s) that will get access to the data produced by OSS-Fuzz, such as bug reports, coverage reports and more stats. Notice the emails affiliated with the project will be public in the OSS-Fuzz repo, as they will be part of a configuration file and they also need to be linked with a Google account to get the detailed reports.

David

### CODE REVIEW

**Review by punchfox — APPROVED** [review]

**Comment by punchfox** [comment]
Hi David,

Thanks for contributing. Let's give this fuzzer thing a try, shall we. :-)

You can set up my email johanl@spotify.com as the one to receive the reports.

This JSON library is not widely used outside Spotify, but we run it in all our clients and also in certain backend services, so running a fuzzer on it definitely seems interesting.

**Comment by punchfox** [comment]
Sorry for the late reply and merge. I'm on parental leave, so didn't check my email for some time.

**Comment by DavidKorczynski** [comment]
> Hi David,
> 
> Thanks for contributing. Let's give this fuzzer thing a try, shall we. :-)
> 
> You can set up my email [johanl@spotify.com](mailto:johanl@spotify.com) as the one to receive the reports.
> 
> This JSON library is not widely used outside Spotify, but we run it in all our clients and also in certain backend services, so running a fuzzer on it definitely seems interesting.

Awesome - we are now waiting for the components to be merged in the OSS-Fuzz repository and once that is merged fuzzing will start :)! 


> Sorry for the late reply and merge. I'm on parental leave, so didn't check my email for some time.

That's wonderful, congratulations and enjoy the rest of your parental leave.

**Pull request merged**
